src/index.js

```diff
@@ -132,7 +126,8 @@ Inspector.prototype = {
       helper.visible = false;
       this.sceneHelpers.add(helper);
       this.helpers[object.uuid] = helper;
-      helper.update();
+      // SkeletonHelper doesn't have an update method
+      if (helper.update) helper.update();
```

Comment:
add brackets to the if body

Reply:
done
I'll fix other ones that eslint reports in the code base as part of #639